src/components/QuickGrid/HeaderColumn.tsx

```diff
@@ -43,7 +48,7 @@ class HeaderColumnInner extends React.PureComponent<IHeaderColumnProps, void> {
                     <div className="header-text">
                         <span
                             key="label"
-                            title={text}
+                            {...tooltipsEnabled ? {title: text} : {} } 
```

Comment:
won't this create a new object each time?

Reply:
You're right. Ty 👍